src/site/content/en/learn-core-web-vitals/crux-and-rum-differences/index.md

```diff
@@ -176,6 +176,8 @@ For security and privacy reasons, top-level pages do not have access to content
 
 CrUX however, being measured by the Chrome browser itself rather than the page, does not have these limitations and so does measure metrics within iframes when reporting Core Web Vitals. This more accurately reflects what the user experiences, but can be another reason for differences for sites making use of iframes.
 
+One concrete example of how this can surface differences between CrUX and RUM data can affect LCP. A `<video>` element's first painted frame can count as an LCP candidate, but embeds for popular video streaming services—such as YouTube, for example—place these elements in an `<iframe>`. CrUX can account for this, as it can access `<iframe>` contents, but RUM solutions cannot.
```

Comment:
Rephrasing and adding a link to the changelog

```suggestion
One concrete example of how this can lead to differences between LCP data in CrUX and RUM is embedded `<video>`. An autoplaying `<video>` element's first painted frame can count as an LCP candidate, but embeds for popular video streaming services—such as YouTube, for example—place these elements in an `<iframe>`. As of [August 2023](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/speed/metrics_changelog/lcp.md), CrUX can account for this, as it can access `<iframe>` contents, but RUM solutions cannot.
```